src/ray/raylet/main.cc

```diff
@@ -236,6 +224,13 @@ int main(int argc, char *argv[]) {
         server->Start();
       }));
 
+  // Initialize stats.
+  const ray::stats::TagsType global_tags = {
+      {ray::stats::JobNameKey, "raylet"},
+      {ray::stats::VersionKey, "0.9.0.dev0"},
+      {ray::stats::NodeAddressKey, node_ip_address}};
+  ray::stats::Init(global_tags, metrics_agent_port, main_service);
```

Comment:
We'd better use another io service for stats.

Reply:
Have you guys measured the overhead before? I assume overhead might be pretty small because it is invoked only every 10 seconds. I can just add TODO now and fix it when I reduce the default period shorter? Do you agree with this?

Reply:
Maybe we can reuse opencensus handler thread or create a separated io service for it. The main service is used for rpc request in metric agent exporter, which will block main process if raylet/nodemanager should handle some requests from worker without waiting or latency. Actually exporter metrics is low priority.